### PR TITLE
 updating order of js scripts and adding a meta tag for viewport

### LINKED ISSUE #11: the browser is zoomed out and the navbar menu doesn't open in mobile screen
1-missing meta tag `<meta name="viewport" content="width=device-width, initial-scale=1">` 

> This means that the browser will (probably) render the width of the page at the width of its own screen. So if that screen is 320px wide, the browser window will be 320px wide, rather than way zoomed out and showing 960px (or whatever that device does by default

2-the order of the scripts " jquery.js,bootstrap.js" is wrong. the jquery.js must be called before bootstrap.js. because that the bootstrap.js require jquery.js to render first to work smoothly  
3-add the `.collapsed` class to the button on the navigation

### PR DESCRIPTION
**Pull request by mohamedragaey**
closes #11.

### CODE REVIEW

File: oshc/templates/base.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <nav class="navbar navbar-default">
       <div class="container-fluid">
         <div class="navbar-header">
-           <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+           <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse">
```
Comment (tapaswenipathak) [inline oshc/templates/base.html:18]: Wouldn't data-toggle and data-target achieve the functionality of collapsing links in the navbar in narrow viewports? How does collapsed class helps in this?

File: oshc/templates/base.html [hunk 2]
```diff
@@ -37,9 +37,9 @@
   <div id="content">
     {% block content %}{% endblock %}
   </div>
+  <script src="http://code.jquery.com/jquery-2.0.3.min.js"></script>
   <script src="{% static "assets/js/bootstrap.js"%}"></script>
```
Comment (tapaswenipathak) [inline oshc/templates/base.html:41]: 👍

**Comment by tapaswenipathak** [comment]
I have added the reference to the issue in the description of your pull request. The issue will get closed as soon as I will merge this one. That's a good way to link pull request and issues.

File: oshc/templates/base.html (hunk 1 above)
Comment (mohamedragaey) [reply oshc/templates/base.html:18]: yes they will achieve the functionality it's a descriptive for the markup

**Pull request merged**